src/test/resources/projects/killed-pod/build.gradle.kts

```
@@ -0,0 +1,40 @@
+plugins {
+    kotlin("jvm") version "2.0.20"
+    id("org.octopusden.octopus.oc-template")
+}
+
+val okdProject = project.findProperty("okd-project") as String?
```

detekt (1.23.8)

[warning] Use the safe cast ('as? String') instead of 'as String?'.

(detekt.potential-bugs.CastToNullableType)